.github/actions/Ubuntu-build/debian/control

```diff
@@ -0,0 +1,36 @@
+Source: imagemagick-7.1.0-16
+Section: graphics
+Priority: optional
+Maintainer: Saurabh Lohe <saurabh.lohe@globallogic.com>
+Build-Depends: autotools-dev, libjbig-dev:amd64,
```

Comment:
same here: ghostscripts cannot be a package dependency but should still be a build dependency